### ISSUE TITLE
Javascript templates [Question]

### ISSUE THREAD

**Issue by alterx** [Azure/azure-functions-cli#225]
I'm using the CLI 2.0 preview (by running `npm i -g azure-functions-core-tools@core`) on MacOS Sierra. I've successfully created a Function App but it seems like I can't create an `HttpTrigger`if I choose `JavaScript` as the language.

Running `func templates list` produces the following output:

```
C# Templates:
  BlobTrigger
  EventGridTrigger
  HttpTrigger
  QueueTrigger
  TimerTrigger

F# Templates:
  BlobTrigger

JavaScript Templates:
  EventGridTrigger
```

Is it possible to create a JS `HttpTrigger`? 
All my functions are created in JS so this basically means I can't test any function offline. Bummer.

**Comment by christopheranderson** [comment]
Hey alterx, two issues:
* JavaScript isn't working out of the box yet, there are some manual steps to enable it. Resolved shortly /cc @mamaso 
* We haven't readded all the JS templates, just that one since we were doing it in a demo. HTTP does work, when you perform the manual steps. In the next week or so, we'll add the JS template back.

TL;DR - this is just a point in time thing. Give us a week and the C# and JS (and maybe the F#? dunno why it is short) list should be equal.

**Comment by alterx** [comment]
Glad to hear that it'll be fixed, debugging my functions online is the worst :( 

In the meantime, I'd love to hear more about those manual steps. If they work, I'd rather do that than `context.log` my way through my functions. 

Thanks for the quick response @christopheranderson  :)

**Comment by alterx** [comment]
This seems to be fixed as of today: https://blogs.msdn.microsoft.com/appserviceteam/2017/09/25/develop-azure-functions-on-any-platform/
